opencog/query/README.md

```diff
@@ -267,21 +268,17 @@ ignored.  This is because constant clauses are "trivial": the
 groundings for them can only be themselves, trivially, and so nothings
 needs be done.
 
-The set of clauses defining a pattern must be connected by means of
-common, shared variables.  A pair of clauses are connected if they
-share a common variable. A set of clauses is connected if there is a
-transitive path through all of them.  The need for this requirement
-was mentioned before: disconnected graphs are always just a trivial
-product of their connected components; thus, it is pointless to
-explicitly support disconnected graphs.
+The clauses are partitioned in to connected components, clauses that
+share directly or indirectly the same variables.
```

Comment:
The old text had a definition of connectivity and transitivity. I think that it is useful for the readers to develop a clear idea that connectivity is transitive (i.e. its the transitive closure of the graph)  Mostly, I just want to use these mathematical terms, since school textbooks already define what the transitive closure of a term algebra is, and I want the reader to just go, "oh yeah, I remember that".  (or look up "transitive closure" on wikipedia, or whatever.)